querier.hh

```diff
@@ -146,7 +155,7 @@ public:
         , _permit(std::move(permit))
         , _range(std::make_unique<const dht::partition_range>(std::move(range)))
         , _slice(std::make_unique<const query::partition_slice>(std::move(slice)))
-        , _reader(ms.make_reader(_schema, _permit, *_range, *_slice, pc, std::move(trace_ptr), streamed_mutation::forwarding::no, mutation_reader::forwarding::no))
+        , _reader(ms.make_reader(underlying_schema(), _permit, *_range, *_slice, pc, std::move(trace_ptr), streamed_mutation::forwarding::no, mutation_reader::forwarding::no))
```

Comment:
Why do we have to take underlying schema here? I thought we're going to pass reversed schema everywhere, no?

Reply:
Once the sstables and memtable patches are reversed yes, but currently all the layers below are reading in forward (native) direction.